.labrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   threshold: 74,
   lint: false,
   globals:
-    'verbose,dryRun,SharedArrayBuffer,Atomics,BigUint64Array,BigInt64Array,BigInt,URL,URLSearchParams,TextEncoder,TextDecoder,queueMicrotask',
+    'verbose,dryRun,SharedArrayBuffer,Atomics,BigUint64Array,BigInt64Array,BigInt,URL,URLSearchParams,TextEncoder,TextDecoder,queueMicrotask,AggregateError,FinalizationRegistry,WeakRef,atob,btoa,AbortController,AbortSignal,EventTarget,Event,MessageChannel,MessagePort,MessageEvent,performance,Blob,BroadcastChannel,structuredClone,DOMException,ReadableStream,ReadableStreamDefaultReader,ReadableStreamBYOBReader,ReadableStreamBYOBRequest,ReadableByteStreamController,ReadableStreamDefaultController,TransformStream,TransformStreamDefaultController,WritableStream,WritableStreamDefaultWriter,WritableStreamDefaultController,ByteLengthQueuingStrategy,CountQueuingStrategy,TextEncoderStream,TextDecoderStream,CompressionStream,DecompressionStream,fetch,FormData,Headers,Request,Response',
```

Comment:
Moving to `@hapi/lab@^24` would allow removing all of those values, but doing so seems to require upgrading `eslint` as `@habi/lab` added a peerDependency on `@hapi/eslint-plugin`, and trying to run the test suite without the plugin gave the error:

```
$ npm run test

> db-migrate@1.0.0-beta.18 pretest
> eslint *.js lib/*.js


> db-migrate@1.0.0-beta.18 test
> lab

node:internal/modules/cjs/loader:933
  const err = new Error(message);
              ^

Error: Failed to load plugin '@hapi' declared in 'BaseConfig': Cannot find module '@hapi/eslint-plugin'
Require stack:
- /Users/mpeveler/code/github/node-db-migrate/__placeholder__.js
Referenced from: BaseConfig
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19)
    at Object.resolve (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2325:46)
    at ConfigArrayFactory._loadPlugin (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3376:33)
    at ConfigArrayFactory._loadExtendedPluginConfig (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3196:29)
    at ConfigArrayFactory._loadExtends (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3117:29)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3058:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3003:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory.create (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2798:16)
    at createBaseConfigArray (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3548:48)
    at new CascadingConfigArrayFactory (/Users/mpeveler/code/github/node-db-migrate/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3688:30)
    at new CLIEngine (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/node_modules/eslint/lib/cli-engine/cli-engine.js:606:36)
    at new ESLint (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/node_modules/eslint/lib/eslint/eslint.js:429:27)
    at Object.<anonymous> (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/lib/modules/coverage.js:20:13)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.get [as coverage] (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/lib/modules/index.js:15:20)
    at Object.exports.run (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/lib/cli.js:39:23)
    at main (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/bin/lab:56:50)
    at Object.<anonymous> (/Users/mpeveler/code/github/node-db-migrate/node_modules/@hapi/lab/bin/lab:60:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/mpeveler/code/github/node-db-migrate/__placeholder__.js' ],
  messageTemplate: 'plugin-missing',
  messageData: {
    pluginName: '@hapi/eslint-plugin',
    resolvePluginsRelativeTo: '/Users/mpeveler/code/github/node-db-migrate',
    importerName: 'BaseConfig'
  }
}
```

Reply:
we did not increase the lab version intentionally, since that requires dropping support for older node versions